course_gather/settings/dev_settings.py

```diff
@@ -1,4 +1,28 @@
+import os
```

Comment:
For dev we should allow all IPs to view the debug toolbar via the SHOW_TOOLBAR_CALLBACK option.

```
DEBUG_TOOLBAR_CONFIG = {
    "SHOW_TOOLBAR_CALLBACK" : lambda request: True,
}
```

Reply:
👍 also added it to local_settings.